platform/linux-generic/odp_packet.c

```diff
@@ -2950,3 +2950,154 @@ odp_packet_reass_partial_state(odp_packet_t pkt, odp_packet_t frags[],
 	(void)res;
 	return -ENOTSUP;
 }
+
+void *odp_packet_buf_head(odp_packet_buf_t pkt_buf)
+{
+	odp_packet_hdr_t *pkt_hdr = (odp_packet_hdr_t *)(uintptr_t)pkt_buf;
```

Comment:
Using a wrapper function for these casts would make the code cleaner.
E.g. `packet_buf_hdr()`

Reply:
using packet_buf_to_hdr() in v15